pinot-common/src/main/java/com/linkedin/pinot/startree/hll/HllConfig.java

```diff
@@ -126,4 +130,12 @@ public int getHllFieldSize() {
   public String toString() {
     return ToStringBuilder.reflectionToString(this, ToStringStyle.SHORT_PREFIX_STYLE);
   }
+
+  public String toJsonString() throws Exception {
```

Comment:
Curious where the callers for these are?

Reply:
Adding hll for pbnj